tests/upgrades/test_client.py

```diff
@@ -23,82 +23,45 @@
 
 import pytest
 from fabric.api import execute
-from nailgun import entities
 from upgrade.helpers.docker import docker_execute_command
 from upgrade_tests.helpers.scenarios import dockerize
 
-from robottelo.api.utils import enable_rhrepo_and_fetchid
 from robottelo.config import settings
-from robottelo.constants import DEFAULT_ARCHITECTURE
 from robottelo.constants import DEFAULT_CV
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
-from robottelo.constants import ENVIRONMENT
 from robottelo.constants import FAKE_0_CUSTOM_PACKAGE
 from robottelo.constants import FAKE_4_CUSTOM_PACKAGE
-from robottelo.constants import PRDS
 from robottelo.constants import REPOS
-from robottelo.constants import REPOSET
 
 
 # host machine for containers
 docker_vm = settings.upgrade.docker_vm
 
 
 @pytest.fixture(scope='module')
-def module_product(default_org):
-    return entities.Product(organization=default_org).create()
-
-
-@pytest.fixture(scope='module')
-def module_lce_library(default_org):
-    """Returns the Library lifecycle environment from chosen organization"""
-    return (
-        entities.LifecycleEnvironment()
-        .search(query={'search': f'name={ENVIRONMENT} and organization_id={default_org.id}'})[0]
-        .read()
-    )
-
-
-@pytest.fixture(scope='module')
-def sat6tools_repo(default_org):
-    """Enable Sat tools repository"""
-    sat6tools_repo_id = enable_rhrepo_and_fetchid(
-        basearch=DEFAULT_ARCHITECTURE,
-        org_id=default_org.id,
-        product=PRDS['rhel'],
-        repo=REPOS['rhst7']['name'],
-        reposet=REPOSET['rhst7'],
-        releasever=None,
-    )
-    sat6tools_repo = entities.Repository(id=sat6tools_repo_id).read()
-    assert sat6tools_repo.sync()['result'] == 'success'
-    return sat6tools_repo
-
-
-@pytest.fixture(scope='module')
-def pre_upgrade_repo(default_org, module_product):
+def pre_upgrade_repo(default_org, module_product, module_target_sat):
     """Enable custom errata repository"""
-    pre_upgrade_repo = entities.Repository(
+    pre_upgrade_repo = module_target_sat.api.Repository(
         url=settings.repos.yum_0.url, product=module_product
     ).create()
     assert pre_upgrade_repo.sync()['result'] == 'success'
     return pre_upgrade_repo
 
 
 @pytest.fixture(scope='module')
-def post_upgrade_repo(default_org, module_product):
+def post_upgrade_repo(default_org, module_product, module_target_sat):
     """Enable custom errata repository"""
```

Comment:
Same here, `default_org` is not in use. Can this be removed?

Reply:
Updated.